src/doc/man/generate-rustc-man.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
This invokes Bash which is not *extremely* portable. I've run this script with `dash` (`/bin/sh` on Debian) and it seems to work either way. Would it make sense to invoke a more standard POSIX shell?